### PR TITLE
DOC: v1.8.0 release notes

### PR DESCRIPTION
**Pull request by ZLLentz**
Just the super cool fuzzy finding feature for now, version numbers are cheap

### CODE REVIEW

File: docs/source/releases.rst [hunk 1]
```diff
@@ -2,6 +2,21 @@
  Release History
 =================
 
+v1.8.0 (2020-10-07)
+===================
+
+Features
+--------
+- Adds bash/fzf-based fuzzy finding of happi items with cooresponding
```
Comment (klauer) [inline docs/source/releases.rst:10]:
```suggestion
- Adds bash/fzf-based fuzzy finding of happi items with corresponding
```

**Review by klauer — APPROVED** [review]
LGTM - and thanks for the releases!

**Comment by klauer** [comment]
Oh, also - careful when the conda-forge PR pops up - need to handle the activation scripts there https://github.com/conda-forge/happi-feedstock/issues/7

**Comment by ZLLentz** [comment]
I saw your fzf happi feedstock PR- thanks for the reminder

**Pull request merged**